examples/chart/teleport/templates/config.yaml

```diff
@@ -77,9 +77,17 @@ data:
       web_listen_addr: {{ .Values.config.listen_addr }}:{{ .Values.ports.proxyweb.containerPort }}
       listen_addr:  {{ .Values.config.listen_addr }}:{{ .Values.ports.proxyssh.containerPort }}
       tunnel_listen_addr:  {{ .Values.config.listen_addr }}:{{ .Values.ports.proxytunnel.containerPort }}
-
+{{- if  .Values.config.teleport.proxy_service.ssh_public_addr }}
+      ssh_public_addr: {{ .Values.config.teleport.proxy_service.ssh_public_addr }}:{{ .Values.service.ports.proxyssh.port }}
```

Comment:
I think we should probably do a conditional check here:
- if `Values.config.teleport.proxy_service.ssh_public_addr` contains `:`, we should use the value as-is and not add a port
- otherwise, we should concatenate `.Values.config.teleport.proxy_service.ssh_public_addr` with `{{ .Values.service.ports.proxyssh.port }}` as we're doing currently.

This is because just about all other examples we give in docs of how to set `public_addr` for various different services have ports in, and if people do that here, they'll end up with `teleport.example.com:3023:3023` in their config files and things will break.

Reply:
set logic to check for : to not put in the port number if a : is in the address